### PR TITLE
Upgrading to use Rails 5

### PR DESCRIPTION
**Pull request by coreydaley**
Upgrading the OpenShift Rails-ex example repository to use Rails 5
Fixes issue #65 

@bparees @luciddreamz Please review

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -1,52 +1,74 @@
 class ArticlesController < ApplicationController
+  before_action :set_article, only: [:show, :edit, :update, :destroy]
 
-  http_basic_authenticate_with name: ENV["APPLICATION_USER"], password: ENV["APPLICATION_PASSWORD"], except: [:index, :show]
```
Comment (luciddreamz) [inline app/controllers/articles_controller.rb:3]: Don't remove authentication from the example. This is part of the template.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -1,21 +1,74 @@
 class CommentsController < ApplicationController
+  before_action :set_comment, only: [:show, :edit, :update, :destroy]
 
-  http_basic_authenticate_with name: ENV["APPLICATION_USER"], password: ENV["APPLICATION_PASSWORD"], only: :destroy
```
Comment (luciddreamz) [inline app/controllers/comments_controller.rb:3]: Same issue, don't remove authentication

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Welcome to OpenShift</title>
```
Comment (luciddreamz) [inline app/views/layouts/application.html.erb:4]: Don't remove the OpenShift customizations to the welcome page

File: config/environments/production.rb [hunk 4]
```diff
@@ -49,17 +47,15 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
-  config.logger =  Logger.new(STDOUT)
```
Comment (luciddreamz) [inline config/environments/production.rb:56]: Moving this to the `if ENV["RAILS_LOG_TO_STDOUT"].present?` block below will disable logging if `RAILS_LOG_TO_STDOUT` is not set in the template. Either revert this change or set the env variable.

**Review by luciddreamz — CHANGES_REQUESTED** [review]

**Comment by luciddreamz** [comment]
Please read through the accompanying README.MD file, deploy the original Ruby on Rails 4 example, and then deploy and compare the updated Ruby on Rails 5 example to ensure both are working as expected before requesting additional review. Also test development mode.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (coreydaley) [reply app/views/layouts/application.html.erb:4]: Fixed

File: config/environments/production.rb (hunk 4 above)
Comment (coreydaley) [reply config/environments/production.rb:56]: Fixed

**Comment by coreydaley** [comment]
@luciddreamz Fixed and tested.  Please review

**Review by luciddreamz — APPROVED** [review]

**Comment by bparees** [comment]
@coreydaley you've deployed this w/ the quickstart templates both with and w/o DBs?

**Comment by bparees** [comment]
sorry, there is no non-DB template for the rails example.  other half of the question stands though.

**Comment by luciddreamz** [comment]
@bparees - Yep, @coreydaley also caught an issue from the original template that was preventing CSS/JS from loading properly. Fixing that now and testing one last time.

**Comment by coreydaley** [comment]
@bparees yes, tested both the postgresql and postgresql-persistent templates.

**Comment by bparees** [comment]
waiting for @luciddreamz's final change.
